app/javascript/app/utils/defaults.js

```diff
@@ -0,0 +1,5 @@
+export const METRIC_OPTIONS = {
```

Comment:
Wouldn't it be better to store this values in `data/constants`, as in `utils` we are mainly exposing functions?

Reply:
Kamil, after this change is done, we are good to merge!

Reply:
@weberjavi Thanks for spotting this, now I see we already have `METRIC_CONSTANT` in `data/constants` so I used this one :)